src/saving_tools.jl

```diff
@@ -200,6 +200,7 @@ Dict{Symbol,Any} with 3 entries:
 function tag!(d::AbstractDict{K,T};
         gitpath = projectdir(), force = false, source = nothing,
         storepatch::Bool = readenv("DRWATSON_STOREPATCH", false),
+        commit_message::Bool = false
```

Comment:
Note that you have added this new functionality but you have not _documented_ it yet. You need to edit the `tag!` function documentation string and describe this new functionality.